python/src/aiconfig/schema.py

```diff
@@ -152,7 +161,7 @@ class Attachment(BaseModel):
     """
 
     # The data representing the attachment
-    data: Any
+    data: Union[AttachmentDataWithStringValue, Any]
```

Comment:
```suggestion
    data: Union[AttachmentDataWithStringValue, str, Any]
```

Technically, an attachment can also be a string, just like `ExecuteResult.data`: `data: Union[OutputDataWithValue, str, Any]`

Reply:
discussed offline: updating to include `str` to be explicit, since we know some models like gemini take in multiple parts or stablediffusion has a 'negative prompt` which is just as much as input. We can iterate as we go about how this looks.

Typescript is defined as JSONValue so no need to add explicit string; Already supported.